tutorials/crictl.md

```diff
@@ -0,0 +1,193 @@
+# CRI-O crictl Tutorial
+
+This tutorial will walk you through the creation of [Redis](https://redis.io/) server running in a [Pod](http://kubernetes.io/docs/user-guide/pods/) using [crictl](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md)
+
+It assumes you've already downloaded and configured `CRI-O`. If not, see [here for CRI-O](setup.md).
+It also assumes you've set up CNI, and are using the default plugins as described [here](contrib/cni/README.md). If you are using a different configuration, results may vary.
+
+## Installation
+
+This section will walk you through installing the following components:
+
+* crictl - The CRI client for testing.
+
+#### Get crictl
+
+```
+go get github.com/kubernetes-sigs/cri-tools/cmd/crictl
+cd $GOPATH/src/github.com/kubernetes-sigs/cri-tools
+make
+make install
```

Comment:
Only the `go get` is be necessary to retrieve `crictl`, so I would recommend in removing the
```
cd $GOPATH/src/github.com/kubernetes-sigs/cri-tools
make
make install
```